client/src/components/primitiveComponents/AlertBoxes.tsx

```diff
@@ -163,4 +163,44 @@ const TwilightEditTimelineAlert = ({
     );
 };
 
-export { TwilightAddEventAlert, TwilightEditTimelineAlert, TwilightEditTextAlert };
+interface TwilightEditCountDialogProps {
+    labelText: string;
+    count: number;
+    onSubmit: (newValue: number) => void;
+}
+
+const TwilightEditCountDialog = ({ labelText, count, onSubmit }: TwilightEditCountDialogProps) => {
+    const [currentCount, setCurrentCount] = useState(count);
+
+    return (
+        <Dialog.Root>
```

Comment:
You might want to put a `onOpen` event on the Root to reset the currentCount state var when the dialog opens and closes.